snyk/managers.py

```diff
@@ -396,3 +397,39 @@ def filter(self, **kwargs: Any):
         post_body = {"filters": filters}
         resp = self.client.post(path, post_body)
         return self.klass.from_dict(self._convert_reserved_words(resp.json()))
+
+
+class IssueSetAggregatedManager(SingletonManager):
+    def all(self) -> Any:
+        return self.filter()
+
+    def filter(self, **kwargs: Any):
+        path = "org/%s/project/%s/aggregated-issues" % (
+            self.instance.organization.id,
+            self.instance.id,
+        )
+        default_filters = {
+            "severities": ["high", "medium", "low"],
+            "exploitMaturity": ["mature", "proof-of-concept", "no-known-exploit", "no-data"],
+            "types": ["vuln", "license"],
+            "priority": {
+                "score": {
+                    "min": 0,
+                    "max": 1000
+                }
+            }
+        }
+
+        post_body = {"filters": default_filters}
+
+        all_filters = list(default_filters.keys()) + ["ignored", "patched"]
+        for filter_name in all_filters:
+            if filter_name in kwargs.keys():
+                post_body["filters"][filter_name] = kwargs[filter_name]
+        
+        for parent_filter in ["includeDescription", "includeIntroducedThrough"]:
```

Comment:
the index variable name `parent_filter` is misleading since it is not used to filter out issues. could you think of a different name that suits here?